test/spec/components/TextArea.spec.js

```diff
@@ -517,6 +521,37 @@ GEHTS
       expect(input.clientHeight).to.be.lessThan(35);
     });
 
+
+    it('should resize when becomes visible', function() {
+
+      // given
+      const result = createTextArea({
+        container,
+        id: 'textarea',
+        autoResize: true
+      });
+
+      const input = domQuery('.bio-properties-panel-input', result.container);
+      const initialHeight = input.clientHeight;
+
+      // when
+      container.style.display = 'none';
+      changeInput(input, 'foo\nbar\nbar\nbar');
+      const hiddenHeight = input.clientHeight;
+
+      // then
+      expect(hiddenHeight).to.be.eq(0);
+
+      // when
+      container.style.display = 'block';
+      const visibleHeight = input.clientHeight;
+
+      // then
+      waitFor(() => {
+        expect(visibleHeight).to.be.greaterThan(initialHeight);
```

Comment:
I think we should make sure that the height as more or less equal to 4 lines as it is not good enough to have a single line of text height.

Reply:
It seems cumbersome to calculate the exact height it should be. but checking for twice the initial size should be enough.